package/endpoint_adapter/src/endpoint_adapter.h

```diff
@@ -31,10 +31,7 @@
 #include <arpa/inet.h>
 
 
-void io_loop_start(int read_fd, int write_fd);
```

Comment:
Changing this function signature will break smoothpose, FYI. @jkretzmer @nicolekelley